addons/website_slides/models/slide_channel.py

```diff
@@ -47,6 +48,36 @@ class ChannelUsersRelation(models.Model):
         )
     ]
 
+    def _compute_next_slide_id(self):
+        self.flush_model()
```

Comment:
this isn't enough, you need to flush
- slide_channel_partner
- slide_slide
- slide_slide_partner

(all the tables used in the query)

Reply:
Added all the tables but just for confirmation - do I need to also add all fields which are used in the query for a flush?

Reply:
no, by default all fields will be flushed :)